src/ray/raylet/scheduling/cluster_task_manager.cc

```diff
@@ -1436,5 +1441,24 @@ ResourceSet ClusterTaskManager::CalcNormalTaskResources() const {
   return ResourceSet(total_normal_task_resources);
 }
 
+uint64_t ClusterTaskManager::MaxRunningTasksPerSchedulingClass(
+    SchedulingClass sched_cls_id) const {
+  auto sched_cls = TaskSpecification::GetSchedulingClassDescriptor(sched_cls_id);
+  double cpu_req = sched_cls.resource_set.GetNumCpusDouble();
```

Comment:
Can we simplify this and early-return INT_MAX if cpu_req=0 or total_cpus=0? I got a bit lost trying to figure out whether the type casting was correct.